dvc/commands/experiments/pull.py

```diff
@@ -3,28 +3,45 @@
 
 from dvc.cli.command import CmdBase
 from dvc.cli.utils import append_doc_link
+from dvc.exceptions import InvalidArgumentError
 from dvc.ui import ui
 
 logger = logging.getLogger(__name__)
 
 
 class CmdExperimentsPull(CmdBase):
+    def raise_error_if_all_disabled(self):
+        if not any(
+            [self.args.experiment, self.args.all_commits, self.args.rev]
+        ):
+            raise InvalidArgumentError(
+                "Either of `experiment`, `--all-commits` "
+                "or `--rev` needs to be set."
```

Comment:
Same as https://github.com/iterative/dvc/pull/7275#pullrequestreview-902546358

Reply:
> Other comments in that PR also apply here.

Reply:
> Other comments in that PR also apply here.

Reply:
Also updated in push.py